projects/storefrontlib/src/recipes/storefront-foundation.module.ts

```diff
@@ -26,6 +27,7 @@ import { ViewConfigModule } from '../shared/config/view-config.module';
     RoutingModule.forRoot(),
     I18nModule.forRoot(),
     CmsModule.forRoot(),
+    QualtricsConfigModule.forRoot(),
```

Comment:
Why do we need `forRoot`? It's a fairly simple addition, let's try to keep it like that, unless there's a good reason.